### PR TITLE
last-window-quits 1.1.3 (new cask)

### PR DESCRIPTION
**Pull request by lawand-dot-io**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/l/last-window-quits.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  name "last-window-quits"
+  desc "Automatically quit apps when their last window is closed"
+  homepage "https://lawand.io/last-window-quits/"
+
+  livecheck do
+    url "https://raw.githubusercontent.com/lawand-dot-io/last-window-quits/main/appcast.xml"
+    strategy :sparkle, &:short_version
+  end
+
+  depends_on macos: ">= :high_sierra"
```
Comment (bevanjkay) [inline Casks/l/last-window-quits.rb:15]:
```suggestion
  auto_update true
  depends_on macos: ">= :high_sierra"
```

I assume based on the presence of Sparkle, that this has an in-app auto updating mechanism?

File: Casks/l/last-window-quits.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+cask "last-window-quits" do
+  version "1.1.3"
+  sha256 "2b6645a0910f27a7021828a44a8744e03db7a7a99b75c3d88ad5fc09c31f29e0"
+
+  url "https://lawand.io/wp-content/uploads/2024/11/last-window-quits-#{version}.zip"
+  name "last-window-quits"
```
Comment (bevanjkay) [inline Casks/l/last-window-quits.rb:6]:
```suggestion
  name "Last Window Quits"

File: Casks/l/last-window-quits.rb (hunk 1 above)
Comment (lawand-dot-io) [reply Casks/l/last-window-quits.rb:15]: Hi. Yes, in-app auto update using sparkle is available

File: Casks/l/last-window-quits.rb (hunk 2 above)
Comment (bevanjkay) [reply Casks/l/last-window-quits.rb:6]: Just the default is fine, thanks.

**Review by bevanjkay — APPROVED** [review]

**Review by bevanjkay — CHANGES_REQUESTED** [review]
Needs a livecheck change...

**Review by bevanjkay — APPROVED** [review]

**Review by khipp — APPROVED** [review]
Thank you for your first contribution to `homebrew-cask`, @lawand-dot-io! 🎉

**Comment by lawand-dot-io** [comment]
> Thank you for your first contribution to `homebrew-cask`, @lawand-dot-io! 🎉

np, happy that my app is added to homebrew 👍

**Pull request merged**